lib/graphql/type/abstract_type.ex

```diff
@@ -1,9 +1,12 @@
-defprotocol GraphQL.AbstractType do
+defprotocol GraphQL.Type.AbstractType do
   @type t :: GraphQL.Type.Union.t | GraphQL.Type.Interface.t
 
   @spec possible_type?(GraphQL.AbstractType.t, GraphQL.Type.ObjectType.t) :: boolean
   def possible_type?(abstract_type, object)
 
+  @spec possible_types(GraphQL.AbstractType.t, GraphQL.Schema.t) :: [GraphQL.Type.ObjectType.t]
+  def possible_types(abstract_type, schema)
```

Comment:
Seems odd that this has a different parameter spec than the other possible_type function.